core/budfox/marketFetcher.js

```diff
@@ -72,7 +72,7 @@ Fetcher.prototype._fetch = function(since) {
   if(++this.tries >= this.limit)
     return;
 
-  this.watcher.getTrades(since, this.processTrades, false);
+  util.retryForever((callback) => this.watcher.getTrades(since, callback, false), this.processTrades);
```

Comment:
I'd rather keep this on the exchange level, if you are retrying inside kraken.js already, what is the need for this?